core/state/statedb.go

```diff
@@ -610,19 +708,13 @@ func (s *StateDB) SubRefund(gas uint64) {
 // Exist reports whether the given account address exists in the state.
 // Notably this also returns true for suicided accounts.
 func (s *StateDB) Exist(addr common.Address) bool {
-	if s.parallel.isSlotDB {
```

Comment:
why delete this

Reply:
"s.parallel.addrStateReadInSlot[addr] = struct{}{}" is already add in getStateObject()